README.md

```diff
@@ -47,8 +47,18 @@ EOF
 $ terraform init
 ```
 
+When submitting PR make sure that it has updates to CHANGELOG.md. Add description of changes under last version with "(unreleased)" mark.
+
+Documenting provider
+--------------------
+Provider documentation is generated automatically when updates are merged into master branch.
+To update documentation contents, please, update "description" field of necessary resource/data_source schema.
```

Comment:
and create/modify templates.

Also place this as the first sentence of the chapter.

Reply:
Fixed